.gitignore

```diff
@@ -2,3 +2,4 @@ build
 .idea
 .gradle
 out
+data/
```

Comment:
There is a gradle task called loadH2data that will create that directory and instantiate an H2 database via the scripts under srct/test/resources/db.  Just need to modify the documentation to reflect.  The migrateSampleData gradle task depends on the loadH2 task.